examples/build_examples.sh

```diff
@@ -17,6 +17,13 @@ else
   PLATFORM=${1}
 fi
 
+if [ -z ${2+x} ]
+then
+  COMPRESSION_ALGORITHM="gzip"
+else
+  COMPRESSION_ALGORITHM=${2}
+fi
+
```

Comment:
I't probably time to document the two optional arguments for this script in the README.md

The enum can have a `Display` impl to be used when invoking `mksquashfs`